assets/prometheus-k8s/kube-rbac-proxy-secret.yaml

```diff
@@ -20,4 +20,9 @@ stringData:
         "user":
           "name": "system:serviceaccount:openshift-monitoring:prometheus-k8s"
         "verb": "get"
+      - "path": "/federate"
+        "resourceRequest": false
+        "user":
+          "name": "system:serviceaccount:openshift-monitoring:telemeter-client"
```

Comment:
this error if telemter-client SA is not allowed:
`auth.go:97] Forbidden (user=system:serviceaccount:openshift-monitoring:telemeter-client, verb=get, resource=, subresource=). Reason: "".`